internal/app/machined/pkg/system/system.go

```diff
@@ -193,7 +193,7 @@ func (s *singleton) Start(serviceIDs ...string) error {
 					msg = strings.ToUpper(msg[:1]) + msg[1:]
 				}
 
-				svcrunner.UpdateState(context.Background(), events.StateFailed, msg)
+				svcrunner.UpdateState(context.Background(), events.StateFailed, "%s", msg)
```

Comment:
what's the formatter doing here?

Reply:
it is the linter which found an issue - `UpdateState` is `printf`-like, so passing `msg` as the first argument is wrong - if it contains `%` instructions, it might lead to invalid result

so we pass `fmt.Printf("%s", v)`